src/devices/Sn74hc595/GpioControllerExtensions.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Device.Gpio;
+using System.Linq;
+
+/// <summary>
+/// Extension methods for GpioController
+/// </summary>
+public static class GpioControllerExtensions
```

Comment:
No namespace here? Also not sure if GpioController extensions belong inside a binding, in my mind this file should instead live under src\devices\Common

Reply:
Ya, I knew that wasn't right. I just decided to wait to hear feedback on the file all-up.